tests/std/tests/P0645R10_text_formatting_parsing/test.cpp

```diff
@@ -51,6 +54,15 @@ struct testing_callbacks {
     constexpr void _On_dynamic_width(_Auto_id_tag) {
         assert(expected_auto_dynamic_width);
     }
+    constexpr void _On_precision(int pre) {
+        assert(pre == expected_precision);
+    }
+    constexpr void _On_dynamic_precision(int id) {
+        assert(id == expected_dynamic_precision);
+    }
+    constexpr void _On_dynamic_precision(_Auto_id_tag) {
+        assert(expected_dynamic_width);

```

Comment:
This appears to be a copy-pasted bug (`expected_dynamic_width` says `width` instead of `precision`, and is an `int` instead of a `bool`). I believe you meant `expected_auto_dynamic_precision` here.